<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_cliente</class>
 <widget class="QDialog" name="dialog_cliente">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consulta</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>481</height>
    </rect>
   </property>
   <widget class="QTableView" name="tb_reporte">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>471</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>41</width>
      <height>53</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbo_reporte">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>283</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>-10</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reportes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_generar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_cerrar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>500</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
